demo/README.md

```diff
@@ -36,7 +36,7 @@ Generally, the latest stable release is safe to use with the latest Scroll SDK c
 
 Now, you can launch all Scroll zkEVM proving services with one command via 
    ```bash
-   SINDRI_VERSION=0.0.3 ./deploy.sh
+   SINDRI_VERSION=0.0.4 ./deploy.sh
```

Comment:
This is in preparation of when we build a new release once this PR is merged. I will test the updates that are in `demo/` and post results.